examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -144,6 +144,7 @@ void PairingCommand::OnStatusUpdate(DevicePairingDelegate::Status status)
     {
     case DevicePairingDelegate::Status::SecurePairingSuccess:
         ChipLogProgress(chipTool, "Secure Pairing Success");
+        ChipLogProgress(chipTool, "CASE establishment successful");
```

Comment:
Why not combine the two logs:

`Secure pairing over CASE succes` and
`Secure pairing over PASE success`

Reply:
I see some tests referencing the string:

src/app/tests/suites/certification/Test_TC_DD_3_5.yaml
src/app/tests/suites/certification/Test_TC_DD_3_7.yaml

however I believe guiding logs by tests (effectively locking in logs) is probably not ideal. If we need to lock in something, we should somehow highlight it as a special string, not make it look like "normal log".

Reply:
I had an explicit request (in https://github.com/project-chip/connectedhomeip/issues/19451) from some of the people writing test plans and scripts to have a log that has "PASE establishment" in it, and to have a separate log with "Pairing success" in it.  I don't know why they want that, but....

Reply:
Resolving, given yes, this was a specific request for log lines from testing.